### PR TITLE
[J4.0] remove backslash from void tags (part 2)

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

This PR is part 2 of https://github.com/joomla/joomla-cms/pull/14000

In HTML5, the backslash is not required on void elements, so this PR remove them from all `<input>` elements.

### Testing instructions

Will need to be code review

### CODE REVIEW

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 319ddc1cca4d67e04a21f80bf4a30c4b2847ffc4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14086">issues.joomla.org/tracker/joomla-cms/14086</a>.</sub>

File: administrator/components/com_content/views/featured/tmpl/default.php [hunk 1]
```diff
@@ -138,7 +138,7 @@
 									<span class="icon-menu"></span>
 								</span>
 									<?php if ($canChange && $saveOrder) : ?>
-										<input type="text" style="display:none" name="order[]" size="5" value="<?php echo $item->ordering; ?>" class="width-20 text-area-order " />
+										<input type="text" style="display:none" name="order[]" size="5" value="<?php echo $item->ordering; ?>" class="width-20 text-area-order ">
```
Comment (yvesh) [inline administrator/components/com_content/views/featured/tmpl/default.php]: One space too much at the end of the class "... "

File: administrator/components/com_installer/views/discover/tmpl/default_item.php [hunk 2]
```diff
@@ -11,8 +11,8 @@
 ?>
 <tr class="<?php echo 'row' . $this->item->index % 2; ?>" <?php echo $this->item->style; ?>>
 	<td>
-			<input type="checkbox" id="cb<?php echo $this->item->index; ?>" name="eid[]" value="<?php echo $this->item->extension_id; ?>" onclick="Joomla.isChecked(this.checked);" <?php echo $this->item->cbd; ?> />
-<!--		<input type="checkbox" id="cb<?php echo $this->item->index; ?>" name="eid" value="<?php echo $this->item->extension_id; ?>" onclick="Joomla.isChecked(this.checked);" <?php echo $this->item->cbd; ?> />-->
+			<input type="checkbox" id="cb<?php echo $this->item->index; ?>" name="eid[]" value="<?php echo $this->item->extension_id; ?>" onclick="Joomla.isChecked(this.checked);" <?php echo $this->item->cbd; ?>>
+<!--		<input type="checkbox" id="cb<?php echo $this->item->index; ?>" name="eid" value="<?php echo $this->item->extension_id; ?>" onclick="Joomla.isChecked(this.checked);" <?php echo $this->item->cbd; ?>>-->
```
Comment (yvesh) [inline administrator/components/com_installer/views/discover/tmpl/default_item.php]: remove the commented line ;)

File: administrator/components/com_content/views/featured/tmpl/default.php (hunk 1 above)
Comment (C-Lodder) [inline administrator/components/com_content/views/featured/tmpl/default.php]: done

File: administrator/components/com_installer/views/discover/tmpl/default_item.php (hunk 2 above)
Comment (C-Lodder) [inline administrator/components/com_installer/views/discover/tmpl/default_item.php]: and done

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on ec68957acda74e3f771be0999efa3eafdaab206c<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14086">issues.joomla.org/tracker/joomla-cms/14086</a>.</sub>

**Pull request merged**